manufacturers/stackups.stanza

```diff
@@ -20,6 +20,11 @@ pcb-material prepreg :
   dielectric-coefficient = 4.26
   name = "PR4 Prepreg 2113/2116"
 
+pcb-material prepreg-2313 :
+  type = Dielectric
+  dielectric-coefficient = 4.05
```

Comment:
Is it useful to annotate units here, or is it obvious what they are?

Reply:
This one is pretty definitive. Also unitless in this instance :P

Reply:
TIL!